update-manifest-build.js

```diff
@@ -0,0 +1,12 @@
+const { resolve } = require("path");
+const { writeFileSync } = require("fs");
+
+const manifestFilePath = resolve(__dirname, "assets", "manifest.json");
+
+const manifest = require(manifestFilePath);
+
+const buildVersion = parseInt(new Date().getTime().toString().slice(-3), 10).toString();
```

Comment:
This convoluted thing ensures that the build will have at most 3 digits (since it can't have a leading 0, we `parseInt` after extracting the last 3), coming from the milliseconds.

Reply:
You know what, the extension manifest is generated on the fly during the build process anyway, so might as well set the build number there: https://github.com/padloc/padloc/blob/v4/packages/extension/webpack.config.js#L90